packages/aws-cdk/lib/api/deploy-stack.ts

```diff
@@ -231,6 +232,11 @@ export async function deployStack(options: DeployStackOptions): Promise<DeploySt
 
   if (await canSkipDeploy(options, cloudFormationStack, stackParams.hasChanges(cloudFormationStack.parameters))) {
     debug(`${deployName}: skipping deployment (use --force to override)`);
+    // if we can skip deployment and we are performing a hotswap, let the user know
+    // that no hotswap deployment happened
+    if (options.hotswap) {
+      print(`\n ${ICON} %s\n`, colors.bold('no hotswap deployment'));
```

Comment:
I feel like this message needs a little polish... We have to make sure it's clear that the deployment isn't happening, because no changes were detected.

Something like:

```suggestion
      print(`\n ${ICON} %s\n`, colors.bold('deployment skipped - no changes were detected (use --force to override)'));
```

Reply:
Updated